crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/manifest_summary.rs

```diff
@@ -39,4 +39,43 @@ pub struct ManifestSummary {
     /// The set of instructions encountered in the manifest that are reserved
     /// and can only be included in the manifest by the wallet itself.
     pub reserved_instructions: Vec<ReservedInstruction>,
+
+    /// The various classifications that this manifest matched against. Note
+    /// that an empty set means that the manifest is non-conforming.
+    pub classification: Vec<ManifestClass>,
+}
+
+use sargon::RetManifestClass as InternalManifestClass;
+
+/// The classification process classifies manifests into classes. The following
+/// are the classes that the Radix Engine Toolkit supports.
+#[derive(Clone, PartialEq, Eq, InternalConversion, uniffi::Enum)]
+pub enum ManifestClass {
+    /// A subintent manifest which satisfies the general rules allowed for in
+    /// general transactions and that includes a [`YieldToParent`] instruction.
+    ///
+    /// [`YieldToParent`]: radix_transactions::manifest::YieldToParent
```

Comment:
can update link to https://github.com/radixdlt/radixdlt-scrypto/blob/main/radix-transactions/src/manifest/ast.rs#L171 later on